packages/ui/src/components/navbar-new/app-sidebar.tsx

```diff
@@ -0,0 +1,219 @@
+                </Link>
+              }
+            />
+          )}
+        </Sidebar.Header>
+        <Sidebar.Content>
+          <Sidebar.Group className="px-4 pt-5">
+            <Sidebar.GroupContent>
+              <Sidebar.Menu>
+                {pinnedMenuItems.map((item, index) => (
+                  <SidebarItem
+                    item={item}
+                    key={index}
+                    handleChangePinnedMenuItem={handleChangePinnedMenuItem}
+                    handleRemoveRecentMenuItem={handleRemoveRecentMenuItem}
+                    handleCustomNav={handleCustomNav}
+                    t={t}
+                  />
+                ))}
+
+                <Sidebar.MenuItem className="cursor-pointer">
+                  <Sidebar.MenuButton asChild onClick={handleMoreMenu}>
+                    <div>
+                      <Icon
+                        name="ellipsis"
+                        size={12}
+                        className="text-icons-4 transition-colors hover:text-primary !w-[12px] !h-[auto]"
+                      />
+                      <span className="text-foreground-3 font-medium transition-colors hover:text-primary">More</span>
+                    </div>
+                  </Sidebar.MenuButton>
+                </Sidebar.MenuItem>
+              </Sidebar.Menu>
+            </Sidebar.GroupContent>
+          </Sidebar.Group>
+
+          {!!recentMenuItems.length && (
+            <Sidebar.Group title="Recent" className="border-t px-4 pt-3">
+              <Sidebar.GroupLabel>Recent</Sidebar.GroupLabel>
+              <Spacer size={2} />
+              <Sidebar.GroupContent>
+                <Sidebar.Menu>
+                  {recentMenuItems.map(item => (
+                    <SidebarItem
+                      isRecent
+                      key={item.id}
+                      item={item}
+                      handleChangePinnedMenuItem={handleChangePinnedMenuItem}
+                      handleRemoveRecentMenuItem={handleRemoveRecentMenuItem}
+                      handleCustomNav={handleCustomNav}
+                      t={t}
+                    />
+                  ))}
+                </Sidebar.Menu>
+              </Sidebar.GroupContent>
+            </Sidebar.Group>
+          )}
+
+          <Sidebar.Group className="border-t px-4 pt-5">
+            <Sidebar.GroupContent>
+              <Sidebar.Menu>
+                {!!currentUser?.admin && (
+                  <Sidebar.MenuItem>
+                    <Sidebar.MenuButton
+                      asChild
+                      className="cursor-pointer"
+                      onClick={() => navigate('/admin/default-settings')}
+                    >
+                      <div>
+                        <Icon
+                          name="account"
+                          size={12}
+                          className="text-icons-4 transition-colors hover:text-primary !w-[12px] !h-[auto]"
```

Comment:
```suggestion
                          className="text-icons-4 transition-colors hover:text-primary"
```

I would prefer to get rid of the `[&>svg]:size-4` class initially in the parent `SidebarMenuButton` component, as this approach does not work in our case, since we pass the svg through the Icon component to which we specify the required sizes, and in the current situation we have to overwrite each time the value set in the parent, so it does not seem to make any sense